src/Pos/Payment/PosPayment.php

```diff
@@ -21,6 +23,10 @@ class PosPayment extends DefaultPayment
 {
     public function pay(Request $request, PaymentTransactionStruct $transaction, Context $context, ?Struct $validateStruct): ?RedirectResponse
     {
-        throw new PaymentMethodNotAvailableException(self::class);
+        if (!Feature::isActive('v10.0.0.0')) {
+            throw new PaymentMethodNotAvailableException(self::class);
+        }
```

Comment:
this doesn't work, we don't have feature flags in paypal - as v10 isn't released yet, just do a break